src/parse-entry.js

```diff
@@ -4,13 +4,23 @@ exports.parseEntry = entry => {
     .split('\n')
 
   const [versionPart, datePart] = title.split(' - ')
-  const [versionNumber] = versionPart.match(/[a-zA-Z0-9.\-+]+/)
+  const [versionNumber] = versionPart.match(/[0-9][a-zA-Z0-9.\-+]+/) || versionPart.match(/unreleased/i)
   const [versionDate] = datePart != null && datePart.match(/[0-9-]+/) || []
+  const status = versionNumber.match(/[-+]+/)
+    ? 'prereleased'
+    : title.match(/\[yanked\]/i)
```

Comment:
I'm not fully following you on this one here. The `versionDate` should still parse as it is happening outside this call. This expression is just looking if `[yanked]` is in the title to denote the status as such.

Reply:
Oh! It is not due to your code 😉 I was just wondering if the action still work with a `[YANKED]` written after the date since I never tested it nor used it before. If you look at the unit tests you'll see no one testing this case, so my first question was "does the action break before even reading this line looking for the yanked word."

Reply:
Ah, yes it will still work because the regex is against the whole title. I'll add additional tests to cover these cases though.

Reply:
Thanks for you work!